src/main/java/com/google/sps/servlets/ChatServlet.java

```diff
@@ -84,17 +98,15 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     @Override
     public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
         final String messageText = (String) request.getParameter(MESSAGE_TEXT_PROPERTY);
-        String groupID = (String) request.getParameter(GROUP_ID_PROPERTY);
+        String groupId = (String) request.getParameter(GROUP_ID_PROPERTY);
         final long timestamp = System.currentTimeMillis();
 
-        // This will be a placeholder until we can coordinate how to pass
-        // groupID from page to page
-        if (groupID == null) {
-            groupID = "123";
+        if (groupId.isEmpty()) {
```

Comment:
Can the groupId no longer be null?

Reply:
no, I'm pulling this parameter from an HTML post form so if no value is given, it just sends an empty string, not null